cardano-cli/src/Cardano/CLI/EraBased/Options/StakePool.hs

```diff
@@ -49,14 +49,14 @@ pStakePoolMetadataHashCmd =
     <$> pPoolMetadataFile
     <*> pMaybeOutputFile
 
-pStakePoolRegistrationCertificiateCmd :: CardanoEra era -> EnvCli -> Maybe (Parser (StakePoolCmds era))
+pStakePoolRegistrationCertificiateCmd :: CardanoEra era -> EnvCli -> Maybe (Parser (StakePoolCmds era)) -- TODO: fix typo in funtion name (extra 'i')
```

Comment:
You can include this typo fix in this pr I think

Reply:
Done :+1: